mamba_gator/handlers.py

```diff
@@ -214,6 +215,26 @@ def post(self):
         self.redirect_to_task(idx)
 
 
+class ExplicitListHandler(EnvBaseHandler):
+    @tornado.web.authenticated
+    def post(self):
```

Comment:
You can directly use the `EnvironmentsHandler.post`. It will do the same if you set `file` instead of `explicitList`. So the REST API stays data oriented.